package.json

```diff
@@ -7,20 +7,22 @@
     "server:migrate": "poetry run manage.py migrate",
     "server:makemigrations": "poetry run manage.py makemigrations && cd backend/api/migrations && git add . && cd ../../osmosewebsite/migrations && git add .",
     "server:seed": "poetry run manage.py seed",
-    "server:run": "poetry run manage.py runserver 8000",
+    "server:run": "python -m poetry run manage.py runserver 8000",
     "server:test": "poetry run manage.py test",
     "server:format": "poetry run black backend",
     "server:lint": "poetry run pylint backend",
     "preaplose:test": "npm run server:seed",
     "aplose:test": "cd frontend && npx cypress open",
     "aplose:run": "cd frontend && npm run dev",
     "osmose:run": "cd website && npm run start",
-
     "reset data": "npm run database:remove && npm run database:create && npm run server:migrate && npm run server:seed",
     "launch": "concurrently --kill-others-on-fail \"npm run database:run && npm run server:migrate && npm run server:run\" \"npm run aplose:run\" \"npm run osmose:run\"",
     "beforeCommit": "npm run \"server:makemigrations\" && npm run \"server:format\" && npm run \"server:lint\""
   },
   "devDependencies": {
     "concurrently": "^8.2.2"
+  },
+  "dependencies": {
+    "html-to-react": "^1.7.0"
```

Comment:
Be careful, this depency is installed for the global project, which shouldn't have this kind of dependencies. It should be installed at the frontend level!